source/3d-secure-2.html.erb

```diff
@@ -0,0 +1,132 @@
+              <a class="govuk-link" href="/govuk-payment-pages" itemprop="item"><span itemprop="name">GOV.UK payment pages</span></a>
+            </li>
+            <li class="sub-navigation__item sub-navigation__item--sub" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+              <a class="govuk-link" href="/api-integration" itemprop="item"><span itemprop="name">API integration</span></a>
+            </li>
+            <li class="sub-navigation__item sub-navigation__item--sub" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+              <a class="govuk-link" href="/take-payments-by-phone-or-post" itemprop="item"><span itemprop="name">Take payments by phone or post</span></a>
+            </li>
+            <li class="sub-navigation__item" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+              <a class="govuk-link" href="/payment-service-provider" itemprop="item"><span itemprop="name">GOV.UK Pay’s Payment Service Provider</span></a>
+            </li>
+            <li class="sub-navigation__item" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+              <a class="govuk-link" href="/direct-debit" itemprop="item"><span itemprop="name">Direct Debit</span></a>
+            </li>
+            <li class="sub-navigation__item" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+              <a class="govuk-link" href="/apple-pay-and-google-pay" itemprop="item"><span itemprop="name">Apple Pay and Google Pay</span></a>
+            </li>
+            <li class="sub-navigation__item" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+              <a class="govuk-link" href="/roadmap" itemprop="item"><span itemprop="name">Roadmap</span></a>
+            </li>
+            <li class="sub-navigation__item sub-navigation__item--active" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+              <a class="govuk-link" href="/security" itemprop="item"><span itemprop="name">Security and compliance</span></a>
+            </li>
+          </ol>
+        </nav>
+      </div>
+
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">Strong Customer Authentication: what it is and how it will work</h1>
+      <p class="govuk-body">Strong Customer Authentication (SCA) will make online payments more secure for users and reduce fraud. It is part of the Payment Service Directive (PSD2) as enforced by the Financial Conduct Authority.</p>
+      <p class="govuk-body">Services that take payments from the EU, Norway, Iceland and Liechtenstein (the European Economic Area, EEA), need to implement SCA by December 2020. Services that only take payments from within the UK need to implement SCA by September 2021.</p>
```

Comment:
would recommend phrasing as "...(otherwise known as the European Economic Area or EEA), must implement..."

What are the exact dates? Maybe we should specify them up front for clarity